multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -18,6 +20,28 @@
 automm_presets = Registry("automm_presets")
 matcher_presets = Registry("matcher_presets")
 
+default_hyperparameter_tune_kwargs = {
+    "searcher": "bayes",
+    "scheduler": "ASHA",
+    "num_trials": 512,
```

Comment:
will this number be too large?

Reply:
I'm not sure. It kind of depends on our search space. How many trials do you think are reasonable?